src/CopySignUpSheetToRoles.js

```diff
@@ -10,65 +10,71 @@
 //    the sign up sheet relies on.
 //  (1-indexed relative to spreadsheet)
 const SIGNUP_START_ROW = 2;
-const SIGNUP_END_ROW = 22;
+const SIGNUP_END_ROW = 23;
 const SIGNUP_START_COL = 2;
-const SIGNUP_END_COL = 9;
+const SIGNUP_END_COL = 7;
```

Comment:
It might be worth seeing if David remembers if there was some other use/goal, but if the roles, minutes, and agendas are generating properly I suppose this is safe.